tests/quantization/test_quantized_ops.py

```diff
@@ -363,8 +363,9 @@ def test_all_arith_ops(
         # Compute the quantized operator result
         quantized_output_vv = q_op(q_inputs_0, q_inputs_1).dequant()
 
-        # Check the R2 of raw output and quantized output
-        check_r2_score(raw_output_vv, quantized_output_vv)
+        if n_bits > 16:
```

Comment:
also, would be better if we change this condition to `if n_bits > 16 and isinstance(operator, (QuantizedDiv, QuantizedMul)):` since it seems to only impact these ops ? or maybe only div ?

Reply:
I updated to have r2 n_bits threshold for each qops